plugin.video.mediathekview/classes/storemysql.py

```diff
@@ -0,0 +1,534 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Leo Moll
+#
+
+# -- Imports ------------------------------------------------
+import string, time
+import mysql.connector
```

Comment:
Instead of dealing with databases directly I'd recommend to consider using some ORM library, for example Peewee: http://docs.peewee-orm.com/en/latest/

Reply:
Regarding the ORM: since the support of two databases cover some different use cases and there are also some specific performance optimisations due to the size and complexity of the database updates, it was a conscious decision not to use an ORM...